src/SFA.DAS.AssessorService.Application.Api/Controllers/OrganisationQueryController.cs

```diff
@@ -112,5 +112,17 @@ public async Task<IActionResult> GetOrganisationByName(string name)
             
             return Ok(Mapper.Map<OrganisationResponse>(organisation));
         }
+        
+        [HttpGet("forContact/{userId}")]
+        public async Task<IActionResult> GetOrganisationForContact(Guid userId)
```

Comment:
The routing for this looks weird for a RESTFUL call. Would this be better in the ContactQueryController?

So the routing would be something like: _/contact/{userId}/organisation_

Reply:
If I was after an organisation, I would expect to find a method to get it in the Organisation controller.  And I think we left RESTful in the dust lonnnnng ago. 😆

Reply:
I would have said it's a Contact's organisation as they own the FK relationship. Anyhow it's works so 👍